### PR TITLE
Added troubleshooting section for download

### LINKED ISSUE #73: Add troubleshooting for Service Download
### Description

Most of the users get stack when downloading. We need to provide in the documentation a workaround to handle that (ssh in the server and manually pull the docker image with `docker pull`).

### LINKED ISSUE #74: Add Contribute GPU Guide for Prem App Swarm Mode
### Description

Include a tutorial step by step on how to contribute to Petals Swarm using Prem App

### PR DESCRIPTION
**Pull request by filopedraz**
### Description

- Added troubleshooting section for a service download issue
- Added instructions for contributing to the swarm from Prem Desktop App
- Closes #73 
- Closes #74

### CODE REVIEW

File: docs/prem-app/troubleshooting/service-download-hanging.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+id: service-download-hanging
+title: Service Download Hanging
+sidebar_label: Service Download Hanging
+sidebar_position: 1
+---
+
+# Service Download Hanging
```
Comment (tiero) [inline docs/prem-app/troubleshooting/service-download-hanging.md:8]:
```suggestion
# Prem Server

## Service Download Hanging
```

Comment (filopedraz) [reply docs/prem-app/troubleshooting/service-download-hanging.md:8]: If we accept these changes, then we need to rename the file too.

Comment (tiero) [reply docs/prem-app/troubleshooting/service-download-hanging.md:8]: The idea is to have one sidebar element for each issue or one page for all?

Comment (filopedraz) [reply docs/prem-app/troubleshooting/service-download-hanging.md:8]: I was thinking of one sidebar element for each issue. Easier to read and find. What do you think?

File: docs/prem-app/network-mode.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+- Select the Model you wanna contribute to. 
+- Select the number of blocks you want to contribute to.
+- Click on `Start` button.
+
+As soon as you Click on the `Start` button, the Prem App will start downloading the blocks from the network and will start contributing to the network. The process requires a coouple of minutes to complete. In the meantime, you will see the following message:
+
+![Network Install](./assets/network-install.png)
+
+When the process is completed, a pop-up will appear with the following message:
```
Comment (nsosio) [inline docs/prem-app/network-mode.md:30]: actually when the pop-up appears it still needs to install requirements the first time (see [this](https://github.com/premAI-io/prem-app/pull/440#discussion_r1374955214)

File: docs/prem-app/network-mode.md
Comment (nsosio) [inline docs/prem-app/network-mode.md:1]: Is it worth to mention that the environment can be cleaned up with the delete button?

File: docs/prem-app/network-mode.md (hunk 2 above)
Comment (filopedraz) [reply docs/prem-app/network-mode.md:30]: Should we specify this? I am not sure.

File: docs/prem-app/network-mode.md
Comment (filopedraz) [reply docs/prem-app/network-mode.md:1]: Good point, but the delete button is not there anymore. Commit has been probably lost in a rebase.

File: docs/prem-app/network-mode.md (hunk 2 above)
Comment (nsosio) [reply docs/prem-app/network-mode.md:30]: Yes maybe it's too much and not necessary

File: docs/prem-app/network-mode.md
Comment (nsosio) [reply docs/prem-app/network-mode.md:1]: I've checked it again today with @tiero and it's there 🤔

Comment (filopedraz) [reply docs/prem-app/network-mode.md:1]: Updated the documentation

**Pull request merged**